src/components/Buttons/CloseTabBtn.tsx

```diff
@@ -13,8 +14,8 @@ const BtnStyle = styled.button`
   z-index: 5;
 `;
 
-function CloseTabBtn({ onClick }) {
-  return <BtnStyle onClick={() => onClick(false)}></BtnStyle>;
+function CloseTabBtn({ onClick }: IOnClick) {
+  return <BtnStyle onClick={() => onClick && onClick(false)}></BtnStyle>;
```

Comment:
이런 경우에는 이렇게 코드를 줄일 수 있어요 :)
```suggestion
  return <BtnStyle onClick={() => onClick?.(false)}></BtnStyle>;
```